### ISSUE TITLE
DotNetCore  Support ?

### ISSUE THREAD

**Issue by NVentimiglia** [facebook-csharp-sdk/facebook-csharp-sdk#368]
Would like dotnetcore support.

**Comment by conniey** [comment]
It appears this library supports Profile 32 (Windows 8.1 and Windows Phone 8.1) which maps to [.NET Standard 1.2](https://docs.microsoft.com/en-us/dotnet/articles/standard/library).  It should be an easy migration.

**Comment by NVentimiglia** [comment]
Yes, i was able to extract what i needed pretty easy. That said, it would be nice to just use a nuget from the official repo in the future.

**Comment by MarkLie** [comment]
Support for .Net Core is also discussed in #363

**Comment by jamiehowarth0** [comment]
I'd also like to argue that due to platform age, .NET 3.5, 4.0, Silverlight 5, and Windows Phone (all platforms) be retired.
Newer releases should be .NET Standard 1.2, 2.0, and the newest .NET 5 standard/framework. This should guarantee compatilibily with Xamarin, UWP, Blazor, and latest cross-platform runtimes with long-term support as well.

**Comment by Mike-E-angelo** [comment]
Anyone got any recommendations on existing NuGet projects that work well with Facebook Graph API?  I am not finding anything obvious/apparent in my searching. 😭

Any assistance/thoughts would be greatly appreciated.

**Comment by Texas-E** [comment]
I'm on .net Core 3.1 and this package is working fine.

**Comment by hynsey** [comment]
> I'm on .net Core 3.1 and this package is working fine.

What platform are you running it on? Does it work on Linux?

**Comment by bcuff** [comment]
> > I'm on .net Core 3.1 and this package is working fine.
> 
> What platform are you running it on? Does it work on Linux?

yeah, it works. we're running it on linux/net3.1 net5.0 & net6.0
